src/orchestrator.ts

```diff
@@ -86,14 +86,51 @@ export class Orchestrator {
             let g = gen.next(undefined);
 
             while (true) {
+
+                if (!(g.value instanceof Task || g.value instanceof TaskSet)) {
+                    if (!g.done) {
+                        // The orchestrator must have yielded a non-Task related type,
+                        // so just return execution flow with what they yielded back.
+                        g = gen.next(g.value);
```

Comment:
So this case handles a user error? For example, they yielded a `5` instead of the result of `context.df.callActivity(...)`?

Reply:
Correct. There is no reason we can't just gracefully return the flow back to their code in this case.